pkgs/by-name/ne/netscanner/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}: let
+  pname = "netscanner";
+  version = "0.4.0";
+in
+  rustPlatform.buildRustPackage {
+    inherit pname version;
+
+    src = fetchFromGitHub {
+      owner = "Chleba";
+      repo = "netscanner";
+      rev = "v${version}";
+      hash = "sha256-E23y37+FTezyTdozFBSGXyiYdr3/iAybbTh2psfhZy4=";
+    };
+
+    cargoLock = {
+      lockFile = ./Cargo.lock;
+    };
+
+    postPatch = ''
+      ln -s ${./Cargo.lock} Cargo.lock
+    '';
+
+    meta = {
+      description = "Network scanner";
```

Comment:
This could be slightly more descriptive, but it's not a blocker.

Reply:
how about
```diff
+  description = "Network scanner with features like WiFi scanning, packetdump and more";
```

Reply:
sounds good to me